.github/workflows/release-note-category.js

```diff
@@ -23,16 +23,29 @@ module.exports = async ({ core, context, github }) => {
   console.log("Available release note category labels:");
   console.log(releaseNoteLabels);
 
-  // Fetch release note category labels applied to this PR
-  const listLabelsOnIssueResp = await github.issues.listLabelsOnIssue({
-    owner,
-    repo,
-    issue_number,
-  });
-  const appliedLabels = listLabelsOnIssueResp.data
-    .map(({ name }) => name)
-    .filter(name => releaseNoteLabels.includes(name));
+  // Fetch the release-note category labels applied on this PR
+  const getAppliedLabels = async () => {
+    const backoffs = [0, 1, 2, 4, 8, 16];
```

Comment:
I think we can remove `0` backoff.

Reply:
@WeichenXu123 We don't need to wait in the initial attempt.

Reply:
The initial attempt might succeed without sleep.